moskito-core/src/main/java/net/anotheria/moskito/core/stats/impl/IntValueHolder.java

```diff
@@ -150,16 +150,16 @@ public int getLastValue() {
 	}
 
 	@Override public void setDefaultValueAsLong(long aValue) {
-		setDefaultValueAsInt((int) aValue);
-	}
+        defaultValue = (int) aValue;
```

Comment:
Within a class you may use the direct manipulation instead of a simple getter. This way you are able to see, that there are no conversions, checks and so on. Some indirections are useful, but this one isn't, because the caller doesn't know, if there are preconditions or other things. What, if the long cannot be converted to an int? In case of a setter you don't know what will happen.